README.md

```diff
@@ -0,0 +1,24 @@
+<h3 align="center">vacay quiz</h3>
+<h4 align="center"><i>Learning Made Fun</i></h4>
+<p align="center"><a href="https://vacay-quiz.netlify.app">Play quizzes »</a></p>
```

Comment:
suggestion - you can remove this link as the same link is at line no 6 (in the overview) which makes sense.

Reply:
I have added this as a call to action to the docs of component library.